hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/FacilityFacade.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2022-2023 Hedera Hashgraph, LLC
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
```

Comment:
Not sure this complies with the "standard" header.  Worth running through spotless to check.

Reply:
That is based on the new Palantir & license rules. I would say the problem is that the imported header is added to new file. This header contains `Copyright (C) ${today.year} Hedera Hashgraph, LLC` but for all other files the old header is still used. Adding @nathanklick

Reply:
Ahh.  If spotless passes, I'd say it's good.